cassandra/search/src/main/java/org/opennms/newts/cassandra/search/CassandraCachePrimer.java

```diff
@@ -84,8 +90,11 @@ public void prime(ResourceMetadataCache cache, Context context) {
                 continue;
             }
 
+            // Let the caches expire before the real TTL to avoid corner-cases and add some margin
+            final long expires = row.getLong("wt") + row.getLong("ttl") * 1000L * 3L / 4L;
```

Comment:
Can we move the `3L / 4L` ratio to a constant?

Reply:
Not in an easy way. As this is integer arithmetic, the order matters: `x * 3 / 4` is different to `x * (3 / 4)`.